packages/react-storage/src/components/StorageBrowser/createStorageBrowser/types.ts

```diff
@@ -118,6 +118,21 @@ export interface StorageBrowserActions {
   custom?: CustomActionConfigs;
 }
 
+export interface StorageBrowserOptions {
+  /**
+   * @description Overrides default file validation called when adding files to be uploaded
+   * @param {File} file — The file to validate
+   * @returns {boolean} — Returns true if `file` is valid, false otherwise
+   * @example
+   * ```tsx
+   * const MAX_FILE_SIZE = 10 * 1000 * 1000 * 1000; // 10 GB in bytes
+   *
+   * const myValidateFile(file: File) => file.size < MAX_FILE_SIZE;
```

Comment:
Syntax is still incorrect. Please see the suggestion [here](https://github.com/aws-amplify/amplify-ui/pull/6555#discussion_r2104803035)